ci/cron/wednesday.yml

```
@@ -46,3 +51,19 @@ jobs:
            --location \
            -d "{\"title\": \"release $RELEASE\", \"head\": \"$BRANCH\", \"base\": \"master\", \"body\": \"This PR has been created by a script, which is not very smart and does not have all the context. Please do double-check that the version prefix is correct before merging.\"}" \
            https://api.github.com/repos/digital-asset/daml/pulls
+
+      setvar "branch" "$BRANCH"
+    name: out
+- job: trigger_ci
+  dependsOn: open_release_pr
+  pool:
+    vmImage: "ubuntu-latest"
+  variables:
+    branch: $[ dependencies.open_release_pr.outputs['out.branch'] ]
+  steps:
+  - checkout: none
+  - bash: |
+      set -euo pipefail
+      az extension add --name azure-devops
+      echo "$(System.AccessToken)" | az devops login --org "https://dev.azure.com/digitalasset"
+      az pipelines build queue --branch $(branch) --definition-name "digital-asset.daml" --org "https://dev.azure.com/digitalasset" --project daml
```

Perhaps put quotes around `$(branch)` so that if it doesn't show up, we get a more meaningful error message?

```suggestion
      az pipelines build queue --branch "$(branch)" --definition-name "digital-asset.daml" --org "https://dev.azure.com/digitalasset" --project daml
```

I want to see an error message along the lines of `invalid branch: ''`, not `could not parse the argument 'digital-asset.daml'`.